runner/engine_test.go

```diff
@@ -688,12 +695,12 @@ func TestRebuildWhenRunCmdUsingDLV(t *testing.T) {
 	go func() {
 		file, err := os.OpenFile("main.go", os.O_APPEND|os.O_WRONLY, 0o644)
 		if err != nil {
-			t.Fatalf("Should not be fail: %s.", err)
+			log.Fatalf("Should not be fail: %s.", err)
```

Comment:
What are the benefits of changing it?

Reply:
To be really honest, I don't know... But my LSP (Gopls) gave me this error and warning:

<img width="695" alt="image" src="https://user-images.githubusercontent.com/4171547/231754756-d0493669-441d-4df5-a4d0-1b72130ec9c9.png">

<img width="584" alt="image" src="https://user-images.githubusercontent.com/4171547/231754811-07200ad0-f83f-45cb-ad39-a3a2ef1db619.png">

But no problem to revert this, just let me know!

Reply:
OK